added: Docker

Liobabrandhoff opened the pull request:
Mit dieser Änderung ist es nun möglich bei der lokalen Ausführung Docker zu verwenden.

Commit:
added: Docker

jona-d01 commented:
Die generierten Bilder sollten nicht mit im Projekt gespeichert bzw. committet werden und mithilfe der `.gitignore` Datei ausgeschlossen werden!

jona-d01 commented on .dockerignore:
```diff
@@ -0,0 +1,5 @@
+/node_modules/
+/uploads/
+Dockerfile
+.dockerignore
```
Die `Dockerfile` und die `.dockerignore`-Datei sollten nicht innerhalb der `.dockerignore` aufgelistet sein!\
Die `.dockerignore`-Datei dient dazu, Dateien und Verzeichnisse aus dem Build-Kontext auszuschließen. Wenn die `Dockerfile` oder die `.dockerignore` selbst in der `.dockerignore` aufgelistet werden, wird Docker sie beim Erstellen des Images ignorieren. Die `.dockerignore` sollte nur die Dateien und Verzeichnisse enthalten, die nicht im Image enthalten sein sollen, wie z. B. temporäre Dateien, Build-Artefakte oder ungenutzte Ressourcen.
```suggestion

```

jona-d01 commented on README.md:
````diff
@@ -19,4 +19,35 @@ Halten Sie die Anwendung, gerade in der Anfangsphase möglichst einfach, schlank
+
+#### Ausführung
+```shell
+docker run -p 8000:8000 --name beiboot-projekt -d docker run -d beiboot-projekt
````
Hier gibts einen kleinen Tippfehler.
Das `docker run -d` am Ende ist doppelt.

```suggestion
docker run -p 8000:8000 --name beiboot-projekt -d beiboot-projekt
```

jona-d01 reviewed (changes requested):
Hier ein paar Änderungsvorschläge, die vor dem Mergen betrachtet werden sollten.

Commit:
removed: dockerignore und Dockerfile in ".dockerignore"
fixed: .gitignore - no generated frames/images
fixed: README - docker run typing error

Liobabrandhoff commented:
@jona-d01 Alle Anmerkungen korrigiert

jona-d01 commented:
Die Frames / Bilder am besten noch aus dem Branch löschen, damit diese nicht versehentlich noch mit gemergt werden 😉
